model/test_results_test.go

```diff
@@ -1067,9 +1061,18 @@ func TestFilterAndSortTestResults(t *testing.T) {
 			},
 			expectedCount: 2,
 		},
+		{
+			name: "TestNameExcludeDisplayNamesFilter",
+			opts: &TestResultsFilterAndSortOptions{
+				TestName:            "B test",
```

Comment:
Could it make sense to give another one of the results a DisplayName that would have matched this filter? (as is the result will be the same whether or not `ExcludeDisplayNames` is specified)

Reply:
Done.